src/createAPI.js

```diff
@@ -5,8 +5,8 @@ import applyMiddleware from './applyMiddleware';
 const createAPI = (
     resources = {},
     middleware = [],
-    APINamespace = '',
-    fetchOptions = {}
+    APINamespace,
+    fetchOptions
```

Comment:
@TimeRaider what do you mean?
We have some default options in `callApiMethod` and you defined the same options higher without any reason. It isn't necessary default options there and now we have wrong behavior. Aren't we?